compute/admin_guide/configure/configure-agentless-scanning.adoc

```diff
@@ -40,7 +40,7 @@ The Monitor and Protect mode has the permission required to enable agentless sca
 Unless you are using a proxy to connect to the Prisma Cloud Console, you must enable auto-assign public IPs on the subnet or security group you use to connect your cloud account to the Prisma Cloud Console. 
 
 [.task]
-=== Configure Agentless Scanning
+=== Configure Agentless Scanning for AWS or GCP Accounts
```

Comment:
@rcaballeromx, where do you plan to add the links for the Azure and AWS templates for permissions?
 
 Azure:

https://redlock-public.s3.amazonaws.com/azure/azureAgentlessPermissions.json

AWS:
https://redlock-public.s3.amazonaws.com/aws/awsAgentlessPermissions.json

Reply:
@Pubs-MV I'm adding the link to the file for Azure in the corresponding steps.
I don't want to add the link to the file for AWS and GCP in the steps, because the flow there is more complex, Instead, I'm providing them with text under the Same Account mode flow for each.
Does that SGTY?